src/address/dto/GetAddress.dto.ts

```diff
@@ -0,0 +1,53 @@
+import { ApiProperty } from '@nestjs/swagger'
+import { Address } from '@prisma/client'
+import { Expose } from 'class-transformer'
+
+export class GetAddressDto implements Address {
+  @ApiProperty({
+    description: 'Unique identifier of an adress.',
+    example: 1
+  })
+    id: number
+
+  @Expose()
+  @ApiProperty({
+    description: 'Number associated with the street',
+    example: '6 ter'
+  })
+    houseNumber: string
+
+  @Expose()
+  @ApiProperty({
+    description: 'The name of the street',
+    example: 'rue Lothaire'
+  })
+    streetName: string
+
+  @Expose()
+  @ApiProperty({
+    description: 'Postcode of the address',
+    example: 67150
+  })
+    zipCode: number
+
+  @Expose()
+  @ApiProperty({
+    description: 'The city or village name of the address.',
+    example: 'Erstein'
+  })
+    city: string
+
+  @Expose()
+  @ApiProperty({
+    description: 'the exact longitude in legal projection.',
+    example: 1044994.79
```

Comment:
Mmmh pk pas utiliser le standard Decimal degree comme mentionné dans https://github.com/compostage-dans-ma-ville/backend/issues/14?

Reply:
Les exemples données ici utilisent le format de api.gouv.fr.

Nous pouvons auss utiliser le standard Decimal degree :)

Reply:
Mdrr les ressources pour stocker, calculer ou convertir des coordonnées géographiques en plusieurs formats, c'est tout un monde https://geodesie.ign.fr/contenu/fichiers/documentation/pedagogiques/TransformationsCoordonneesGeodesiques.pdf

Reply:
:white_check_mark: in d66dee9